README.md

```diff
@@ -1,119 +1,189 @@
-[![PWC](https://img.shields.io/endpoint.svg?url=https://paperswithcode.com/badge/danish-fungi-2020-not-just-another-image/image-classification-on-df20)](https://paperswithcode.com/sota/image-classification-on-df20?p=danish-fungi-2020-not-just-another-image)
+[![PWC](https://img.shields.io/badge/WACV-2022-blue)](https://wacv2022.thecvf.com/)
+[![PWC](https://img.shields.io/badge/Proceedings-CVF-red)](https://openaccess.thecvf.com/content/WACV2022/html/Picek_Danish_Fungi_2020_-_Not_Just_Another_Image_Recognition_Dataset_WACV_2022_paper.html)
+[![PWC](https://img.shields.io/badge/Pretrained--Models-HuggingFace-blue)](https://huggingface.co/BVRA)
 
-[![PWC](https://img.shields.io/endpoint.svg?url=https://paperswithcode.com/badge/danish-fungi-2020-not-just-another-image/image-classification-on-df20-mini)](https://paperswithcode.com/sota/image-classification-on-df20-mini?p=danish-fungi-2020-not-just-another-image)
 
-# News
-- Metrics slightly updated! Retrained with PyTorch NGC Docker Container 20.07 and on Ampere GPUs only (3080 / 3090)
-- EXIF metadata available! You can read it dirrectly from the images.
-
-# Danish Fungi 2020 - Not Just Another Image Recognition Dataset
+# 🍄 Danish Fungi – Not Just Another Image Recognition Dataset
 
 By [Lukas Picek](https://sites.google.com/view/picekl) et al. 
 [MAIL](mailto:lukaspicek@gmail.com?subject=[GitHub]%20DanishFungi2020%20Project)
 
-## Introduction
-
-Supplementary material to:
-Danish Fungi 2020 - Not Just Another Image Recognition Dataset 
-
-In order to support research in fine-grained plant classification and to allow full reproducibility of our results, we share the Training Logs and Trained scripts.
-- The Images, Checkpoints and Metadata are not included based on size constrains and will be published after the review.
-
-## Training Data
+## Description
+
+Danish Fungi 2020 (DF20) is a new dataset for fine-grained visual categorization. The datasets is 
+constructed from observations submitted to the Atlas of Danish Fungi and is unique in
+(i) its taxonomy-accurate labels with little to no label errors,
+(ii) highly unbalanced long-tailed class distribution,
+(iii) rich observation metadata about surrounding environment, location, time and device.
+DF20 has zero overlap with ImageNet, allowing unbiased comparison of models fine-tuned from publicly 
+available ImageNet checkpoints. The proposed evaluation protocol enables testing the ability to 
+improve classification using metadata – e.g. precise geographic location, habitat, and substrate,
+facilitates classifier calibration testing, and finally allows to study the impact of the device 
+settings on the classification performance.
+
+![Species Similarities and differences](figures/fungi_samples.png)
+
+## News
+- We made a new train/test split based on ObservationIDs where data from the same observation do not occur in the test set. 
```

Comment:
maybe we might want to add dates to news so that people now if sth has changed since they last used it